src/adapter/actions/LegacyContextMenuAction.js

```diff
@@ -32,6 +32,23 @@ export default class LegacyContextMenuAction {
         this.LegacyAction = LegacyAction;
     }
 
+    invoke(objectPath) {
```

Comment:
@akhenry Is this a functional change or just reordering code?

Reply:
Just a reorder. It made more sense to me to have the function that defines what the Action does above the function that defines which objects it applies to. It's a trivial thing that creates unnecessary noise in the diff :)

Reply:
roger, makes sense.  just saving myself from having to manually diff these lines to see if anything changed.